### PR TITLE
Release notes for 1.0

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

Initial release notes for the 1.0 release. As usual, any mistakes by me and awesome artwork by Máirín.

### Screenshot/screencast of this PR

Screenshot 2023-05-17 at 6 25 35 PM

### What issues does this PR fix or reference?

N/A

### CODE REVIEW

**Comment by jeffmaury** [comment]
Make good to celebrate the community to list all contributors since first commit.

File: website/blog/2023-05-17-release-1.0.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+![Featured extensions](img/podman-desktop-release-1.0/featured-extensions.png)
+
+Have another idea for extending Podman Desktop? We would love to hear from you or see
+what you can create with the [Extension documentation](../docs/extensions).
+
+### Podman Machine as Root
+
+When creating a Podman machine you can now decide to run as root [#2427](https://github.com/containers/podman-desktop/pull/2427).
+
```
Comment (themr0c) [inline website/blog/2023-05-17-release-1.0.md:84]:
```suggestion

This is a prerequisite in some scenarios, such as running Kind on Windows.

```

File: website/blog/2023-05-17-release-1.0.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+Tired of clicking +, +, +? Us too. Numeric preferences are now editable so
+you can directly enter the value you want
+[#2368](https://github.com/containers/podman-desktop/pull/2368).
+
+![Editing numbers](img/podman-desktop-release-1.0/edit-number.png)
+
+#### Navigation tooltips
+
+We like our new naviagtion bar, but the tooltips were slow to appear. This made it harder for new
```
Comment (themr0c) [inline website/blog/2023-05-17-release-1.0.md]: typo

```suggestion
We like our new navigation bar, but the tooltips were slow to appear. This made it harder for new
```

Comment (deboer-tim) [reply website/blog/2023-05-17-release-1.0.md]: Fixed.

File: website/blog/2023-05-17-release-1.0.md (hunk 1 above)
Comment (deboer-tim) [reply website/blog/2023-05-17-release-1.0.md:84]: Added, thanks.

**Comment by deboer-tim** [comment]
@jeffmaury Added a full list of contributors, let me know if you had something else in mind.

**Comment by jeffmaury** [comment]
> @jeffmaury Added a full list of contributors, let me know if you had something else in mind.

Can we get them sorted alphabetically ?

**Comment by deboer-tim** [comment]
Sorted the contributors.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by mairin** [comment]
if you want to use a more general 1.0 image i made this one too, up2u

penup_20230519_230741

**Comment by deboer-tim** [comment]
I didn't have a chance to update the screenshot at the top yet, but if you look at the last commit you'll see I switched to that image.

**Review by benoitf — APPROVED** [review]

**Pull request merged**